src/main/java/com/iridium/iridiumteams/commands/FlyCommand.java

```diff
@@ -36,6 +42,12 @@ public boolean execute(U user, T team, String[] args, IridiumTeams<T, U> iridium
             return false;
         }
 
+        if(!iridiumTeams.getTeamManager().isInTeam(team, player.getLocation())) {
+            player.sendMessage(iridiumTeams.getMessages().notInTeamLand
+                    .replace("%prefix%", iridiumTeams.getConfiguration().prefix));
+            return false;
+        }
+
```

Comment:
This isnt needed, its done inside the canFly method

Reply:
ive been staring at this method for an hour, you're right, how did i miss that lol